### PR TITLE
[civ2] build ci base test and ci base build images

### PR DESCRIPTION
**Pull request by aslonnie**
build all from source, no prebuilt-bases.

### CODE REVIEW

File: ci/docker/base.test.Dockerfile [hunk 1]
```diff
@@ -44,14 +52,11 @@ ENV LC_ALL=en_US.utf8
 ENV LANG=en_US.utf8
 RUN echo "ulimit -c 0" >> /root/.bashrc
 
-# Setup Bazel caches
-RUN echo "build --remote_cache=${REMOTE_CACHE_URL}" >> /root/.bazelrc
-
```
Comment (aslonnie) [inline ci/docker/base.test.Dockerfile:49]: `ci/env/install-bazel.sh` will always fully rewrite the `.bazelrc` file, so there is actually no point to write it here.

**Comment by aslonnie** [comment]
all the base images builds. civ2 not blocks for 24 minutes.

there is one more layer that needs to be built to be able to run tests.

File: ci/docker/base.test.wanda.yaml [hunk 2]
```diff
@@ -6,5 +6,6 @@ srcs:
   - .bazeliskrc
 build_args:
   - REMOTE_CACHE_URL
```
Comment (can-anyscale) [inline ci/docker/base.test.wanda.yaml:8]: do we still need this?

File: ci/docker/base.test.wanda.yaml [hunk 3]
```diff
@@ -6,5 +6,6 @@ srcs:
...
+  - BUILDKITE_BAZEL_CACHE_URL
```
Comment (can-anyscale) [inline ci/docker/base.test.wanda.yaml:9]: is the value of this default inside wanda?

**Review by can-anyscale — APPROVED** [review]

File: ci/docker/base.test.wanda.yaml (hunk 3 above)
Comment (aslonnie) [reply ci/docker/base.test.wanda.yaml:9]: no, but it is set by the `rayci` command, the value is from the rayci pipeline config.

File: ci/docker/base.test.wanda.yaml (hunk 2 above)
Comment (aslonnie) [reply ci/docker/base.test.wanda.yaml]: nope. good catch. removed.

**Comment by aslonnie** [comment]
going in; let the world explode~

**Pull request merged**